test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/DotnetTestHostManagerTests.cs

```diff
@@ -639,6 +641,11 @@ public void GetTestHostProcessStartInfoShouldIncludeTestHostPathNextToTestRunner
         StringAssert.Contains(startInfo.Arguments, $"--runtimeconfig \"{expectedRuntimeConfigPath}\"");
     }
 
+#endif
+
+    // TODO: Not sure why but this worked before on .NET451, but now I can't get it to work anywhere. Assembly.GetEntryAssembly().Location is null because of running in app domain.
```

Comment:
It did not.
```suggestion
    // TODO: This assembly was previously compiled as net472 and so it was skipped and only ran as netcoreapp2.1. This fails in test, but works in code that is not isolated in appdomain. Might be worth fixing because we get one null here, and another in DotnetTestHostManager.
    // Assembly.GetEntryAssembly().Location is null because of running in app domain.
```